docs/running_a_service.md

```diff
@@ -0,0 +1,246 @@
+<!--[metadata]>
++++
+title = "Running a Notary service"
+description = "Run your own notary service to host arbitrary content signing."
+keywords = ["docker, notary, notary-server, notary server, notary-signer, notary signer"]
+[menu.main]
+parent="mn_notary"
+weight=4
++++
+<![end-metadata]-->
+
+# Run a Notary service
+
+This document is for anyone who wants to [run their own Notary
+service](running_a_service.md) (such as those who want to use Notary with a
+private Docker registry). Running a Notary service requires that you are already
+familiar with using [Docker Engine](https://docs.docker.com/engine/userguide/)
+and [Docker Compose](https://docs.docker.com/compose/overview/).
+
+## Run a service for testing or development
+
+The quickest way to spin up a full Notary service for testing and development
+purposes is to use the Docker compose file in the
+[Notary project](https://github.com/docker/notary).
+
+```plain
+$ git clone https://github.com/docker/notary.git
+$ cd notary
+$ docker-compose up
+```
+
+This will build the development Notary server and Notary signer images, and
+start up a containers for the Notary server, Notary signer, and the MySQL
+database that both of them share.  The MySQL data is stored in a volume.
+
+Notary server and Notary signer communicate over mutually authenticated TLS
+(using the self-signed testing certs in the repository), and Notary server
+listens for HTTPS traffic on port 4443.
+
+By default, this development Notary server container runs with the testing
+self-signed TLS certificates. In order to be able to successfully connect to
+it, you will have to use the root CA file in `fixtures/root-ca.crt`.
+
+For example, to connect using OpenSSL:
+
+```bash
+$ openssl s_client -connect <docker host>:4443 -CAfile fixtures/root-ca.crt -no_ssl3 -no_ssl2
+```
+
+To connect using the Notary Client CLI, please see [Getting Started](getting_started.md)
+documentation.
+
+The self-signed certificate's subject name and subject alternative names are
+`notary-server`, `notaryserver`, and `localhost`, so if your Docker host not
+on localhost (for example if you are using Docker Machine), you'll need to
+update your hosts file such that the name `notary-server` is associated with
+the IP address of your Docker host.
+
+## Advanced configuration options
+
+Both the Notary server and the Notary signer take
+[JSON configuration files](reference/index.md). Pre-built images, such as
+the [development images above](#notary-service-temp) provide these configuration
```

Comment:
this link doesn't seem to be working for me, it seems like we should link to line 20 in this doc?